charts/dapr/README.md

```diff
@@ -262,3 +263,12 @@ kubectl port-forward dapr-operator-5c99475ffc-m9z9f 40000:40000 -n dapr-system
 ```
 helm install dapr dapr/dapr --namespace dapr-system --set global.nodeSelector.myLabel=myValue --wait
 ```
+
+## Example of seamless integration with cert-manager to manage sentry certificates and mTLS
```

Comment:
I don't think this section belongs here. Instead, can you please open an issue in dapr/docs (or directly a PR) to document this?

In the docs, it should go this way:

1. Explain the options that are now available.
2. Explain what they can be used for, in general terms (ie. "override the file name - but not path - of the certs that are loaded within the container)
3. Lastly, explain how this is relevant to cert-manager users

Reply:
I removed that paragraph, as you suggest I will open an issue for docs, my purpose was add seamless integration with cert-manager as said in #4428